comp_loinc/build.py

```diff
@@ -126,7 +127,7 @@ def merge_owl(
     TODO: Consider removing the files created from this point each time this code executes e.g. any file with 'merge_*'
     """
     files = [os.path.join(owl_directory, str(x)) for x in os.listdir(owl_directory) if ".owl" in str(x)]
-    subprocess.call(["./robot", "merge", "-i"] + " -i ".join(files).split() + ['-o', output])
+    subprocess.call([ROBOT_BIN_PATH, "merge", "-i"] + " -i ".join(files).split() + ['-o', output])
```

Comment:
Bugfix: build.py: robot path is now an absolute path. Solves problem where path did not resolve based on where the script was being called from. (this error came up during my tests)